packages/grpc-js-core/src/client.ts

```diff
@@ -8,28 +8,32 @@ import {ChannelCredentials} from './channel-credentials';
 import {Status} from './constants';
 import {Metadata} from './metadata';
 
+// This symbol must be exported (for now).
+// See: https://github.com/Microsoft/TypeScript/issues/20080
+export const kChannel = Symbol();
+
 export interface UnaryCallback<ResponseType> {
   (err: ServiceError|null, value?: ResponseType): void;
 }
 
 export class Client {
-  private readonly channel: Channel;
+  private readonly [kChannel]: Channel;
```

Comment:
Currently investigating whether this is needed (I changed this while experimenting with the ServiceClient class)

Reply:
Ok, now I remember -- this is necessary because of this line.

https://github.com/grpc/grpc-node/pull/206/files#diff-bef77206fce0848d6ef29377963c0a10R115